runner/src/main/openapi/runner.yaml

```diff
@@ -1,8 +1,12 @@
 openapi: 3.0.3
 
 info:
-  title: ""
-  version: ""
+  title: Cosmo Tech Runner Manager API
+  description: Cosmo Tech Runner Manager API
+  version: 1.0.0-SNAPSHOT
+
+servers:
+  - url: 'http://localhost:8080'
```

Comment:
Do we need that back? If I recall correctly, I removed all these from the sub-module openapi specs as they will be overwritten during the merge to generate the final spec. Before we had some warnings during the merge about overridden or ignored data that was fixed by not putting `info` and `servers` config in the split specs.